components/script/dom/bindings/codegen/Configuration.py

```diff
@@ -336,6 +342,10 @@ def isGlobal(self):
 
 
 # Some utility methods
+def getModuleFromObject(object):
+    return object.location.filename().split('/')[-1].split('.webidl')[0] + 'Binding'
```

Comment:
This could also be written:

```python
import os.path as path

path.splitext(path.basename(object.location.filename()))[0]
```

https://docs.python.org/2/library/os.path.html#os.path.basename

https://docs.python.org/2/library/os.path.html#os.path.splitext

The main thing concern here is the `.split('/')` which *might* cause issues on Windows